evap/staff/views.py

```diff
@@ -1461,11 +1447,12 @@ def evaluation_person_management(request, evaluation_id):
                 delete_import_file(request.user.id, import_type)
             elif "copy" in operation:
                 copy_form.evaluation_selection_required = True
-                if copy_form.is_valid():
-                    import_evaluation = copy_form.cleaned_data["evaluation"]
-                    importer_log = import_persons_from_evaluation(
-                        import_type, evaluation, test_run=False, source_evaluation=import_evaluation
-                    )
+                if not copy_form.is_valid():
+                    raise SuspiciousOperation("Invalid copy form")
```

Comment:
What happens when I don't select an evaluation from the dropdown? (Possibly with JS disabled?)

Reply:
We get into the same problem even with JS enabled